pxr/base/tf/pathUtils.cpp

```diff
@@ -66,7 +66,13 @@ _ExpandSymlinks(const std::string& path)
     // and the remaining part of the path.
     std::string::size_type i = path.find_first_of("/\\");
     while (i != std::string::npos) {
-        const std::string prefix = path.substr(0, i);
+        std::string prefix = path.substr(0, i);
+        // If the prefix is "X:", this will access the "current" directory on
+        // drive X, when what we really want is the root of drive X, so append
+        // a backslash.
+        if (prefix.at(i-1) == ':') {
+            prefix.push_back('\\');
```

Comment:
I would not use backslash unnecessarily. Forward slash works.

Reply:
I thought about that a bit too. But this is pure Windows code, and the returned symlinks all have backslashes in them... In the end I decided it didn't matter. But I'm happy either way.